draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -320,6 +318,11 @@ implementations that process certificates and CRLs using ML-DSA MUST
 recognize the corresponding OIDs. Encoding rules for ML-DSA signature
 values are specified in {{oids}}.
 
+When creating and verifying signatures in scope of this specification
```

Comment:
I like this, but I suggest to move it in the paragraph that mentions how these signatures are computed. 

> The signatureValue field contains the corresponding ML-DSA signature computed upon the ASN.1 DER encoded tbsCertificate/tbsCertList {{RFC5280}}.

Reply:
Like this?